net-mail/mboxgrep/mboxgrep-0.7.9-r3.ebuild

```
@@ -0,0 +1,49 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit autotools
+
+DESCRIPTION="Grep for mbox files"
+SRC_URI="mirror://sourceforge/mboxgrep/${P}.tar.gz"
+HOMEPAGE="https://datatipp.se/mboxgrep/"
+
+SLOT="0"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="dmalloc"
+
+RDEPEND="
+	app-arch/bzip2
+	dev-libs/libpcre
+	sys-libs/zlib
+	dmalloc? ( dev-libs/dmalloc )
+"
+DEPEND="
+	${RDEPEND}
+"
+PATCHES=(
+	"${FILESDIR}"/${P}-_DEFAULT_SOURCE.patch
+	"${FILESDIR}"/${P}-fno-common.patch
+	"${FILESDIR}"/${P}-ldflags.patch
+	"${FILESDIR}"/${PN}-0.7.9-musl-missing-strcmp.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_with dmalloc no yes)
```

Hmm I don't remember if this syntax changed between EAPIs 6 and 8, but I'm getting:
```
 * QA Notice: Unrecognized configure options:
 * 
 * 	--without-no
 * 	--without-no
```
`./configure --prefix=/usr --build=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu --mandir=/usr/share/man --infodir=/usr/share/info --datadir=/usr/share --sysconfdir=/etc --localstatedir=/var/lib --datarootdir=/usr/share --docdir=/usr/share/doc/        mboxgrep-0.7.9-r3 --htmldir=/usr/share/doc/mboxgrep-0.7.9-r3/html --libdir=/usr/lib64 --without-no`

Something like
```suggestion
	$(use_with dmalloc dmalloc $(usex dmalloc yes no))
```
should work though.